### DISCUSSION TITLE
Calendar Drag&Drop not working

### DISCUSSION THREAD

**Discussion by carstenlehmann** [kimai/kimai#3823] [Questions & Answers] [answered]
Hello,

I am attempting to utilize the drag and drop feature within the calendar in the Kimai software, however I am having difficulty in doing so. I am currently using Version 1.30.5 of the software and am unsure if I am missing a setting that would enable this feature. Could you please provide any assistance in this matter?

Furthermore, I also do not see the small symbol beneath a time entry to simply change the time by dragging and dropping. Neither the Team Leader role nor the Super-Admin can use drag & drop in the calendar.

![image](https://user-images.githubusercontent.com/55583074/216939899-14716e46-19ab-44cc-bc36-5bd6f52abf7e.png)

**Comment by kevinpapst** [comment, marked as answer]
As the calendar works otherwise, this is likely not a Javascript problem.

But Drag & Drop is globally deactivated if the time-tracking mode does not allow to edit the start-time (e.g. punch mode).

**Reply by carstenlehmann** [reply]
Amazing! Thank you soooo much!
